src/web3-wrapper/rpc/rpcInfo.ts

```diff
@@ -0,0 +1,6 @@
+export class RpcInfo {
+  url: string;
+  weight: number;
+  chainId?: string;
+  isArchiveNode?: boolean;
```

Comment:
can we check against the model in the db and put the rest of the fields in here as well?